registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/AnonymousProfileServiceImpl.java

```diff
@@ -56,5 +102,150 @@ public void saveAnonymousProfile(String regId, String processStage, String profi
 					"AnonymousProfileServiceImpl::saveAnonymousProfile()::exit");
 
 	}
+	
+	@Override
+	public String buildJsonStringFromPacketInfo(BiometricRecord biometricRecord, Map<String, String> fieldMap,
+			Map<String, String> metaInfoMap, String statusCode, String processStage)
+			throws JSONException, ApisResourceAccessException, PacketManagerException, IOException {
+
+		AnonymousProfileDTO anonymousProfileDTO = new AnonymousProfileDTO();
+		anonymousProfileDTO.setProcessName(
+				getFieldValueFromMetaInfo(metaInfoMap, JsonConstant.METADATA, JsonConstant.REGISTRATIONTYPE));
+		anonymousProfileDTO.setProcessStage(processStage);
+		anonymousProfileDTO.setStatus(statusCode);
+		String date = metaInfoMap.get(JsonConstant.CREATIONDATE);
+		anonymousProfileDTO.setDate(DateUtils.getTimeFromDate(
+				DateUtils.parseToDate(date, ANONYMOUS_PROFILE_DATETIME_PATTERN), ANONYMOUS_PROFILE_DATETIME_PATTERN));
+
+		String dateArr[] = fieldMap.get("dateOfBirth").split("/");
+		anonymousProfileDTO.setYearOfBirth(dateArr[0]);
+		anonymousProfileDTO.setGender(getLanguageBasedValueForSimpleType(fieldMap.get(MappingJsonConstants.GENDER)));
+
+		String zone = getLanguageBasedValueForSimpleType(fieldMap.get("zone"));
+		String postalCode = fieldMap.get("postalCode");
+		anonymousProfileDTO.setLocation(Arrays.asList(zone, postalCode));
+		anonymousProfileDTO.setPreferredLanguages(Arrays.asList(fieldMap.get(MappingJsonConstants.PREFERRED_LANGUAGE)));
+		List<String> channels = new ArrayList<String>();
+		if (fieldMap.containsKey(MappingJsonConstants.PHONE) && fieldMap.get(MappingJsonConstants.PHONE) != null) {
+			channels.add(MappingJsonConstants.PHONE);

```

Comment:
read phone value from mapping json . use below code -

JSONObject regProcessorIdentityJson = utility.getRegistrationProcessorMappingJson(MappingJsonConstants.IDENTITY);
String email = JsonUtil.getJSONValue(JsonUtil.getJSONObject(regProcessorIdentityJson, MappingJsonConstants.EMAIL),MappingJsonConstants.VALUE);
String phone = JsonUtil.getJSONValue(JsonUtil.getJSONObject(regProcessorIdentityJson, MappingJsonConstants.PHONE),MappingJsonConstants.VALUE);

Reply:
i have used similar logic what jyoti has used... bcoz utility class is not accesible to status service impl...